index.html

```diff
@@ -321,15 +321,109 @@ <h2 class="title is-size-2 mb-6">Editar categoría</h2>
             <div class="columns is-centered is-mobile">
                 <div class="column is-8-widescreen is-9-desktop is-12-tablet">
                     <article class="box">
-                        <h2 class="title is-size-2">Reportes</h2>
-                        <div class="columns is-centered my-6 py-6">
+                        <h2 class="title is-size-2 has-text-weight-bold">Reportes</h2>
+                        <!-- <div id= "sin-reportes" class="columns is-centered my-6 py-6">
```

Comment:
y eso.. no lo comentes. Ponele un class="is-hidden" porque es lo que va a ir si no aparece ninguna operación
me refiero al div con el id="sin-reportes"

Reply:
Ese lo comente porque no esta funcional la parte de si se ve el html o se ve la imagen de sin reportes. Despues lo descomento